### PR TITLE
This PR adds data and metadata ops after upgrading secondary zone 

### PR DESCRIPTION
**Pull request by viduship**
This is to add data and metadata ops after upgrading the secondary zone first

bug [2327402](https://bugzilla.redhat.com/show_bug.cgi?id=2327402)

http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-BWPZ5K

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/squid/rgw/tier-1_rgw_ms_upgrade_61GA_to_8x_latest.yaml [hunk 1]
```diff
@@ -375,7 +388,7 @@ tests:
 
   - test:
       clusters:
-        ceph-pri:
+        cephi-sec:
```
Comment (mkasturi18) [inline suites/squid/rgw/tier-1_rgw_ms_upgrade_61GA_to_8x_latest.yaml:391]: please fix typo

**Review by mkasturi18 — CHANGES_REQUESTED** [review]

File: suites/squid/rgw/tier-1_rgw_ms_upgrade_61GA_to_8x_latest.yaml (hunk 1 above)
Comment (viduship) [reply suites/squid/rgw/tier-1_rgw_ms_upgrade_61GA_to_8x_latest.yaml]: done.

**Review by anrao19 — APPROVED** [review]
LGTM

**Comment by TejasC88** [comment]
LGTM

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**